README.md

```diff
@@ -6,20 +6,30 @@ Work with NuPIC results interactively in a nice web UI. This app is designed to
 
 ## Requirements
 
-* The first line in the CSV must be a header, containing the names of the fields
-* There must be a field labeled `timestamp` in the list of fields.
-* The data in the `timestamp` field can be either a number or a string. If it is a string, it will attempt to parse the date. To ensure that the app can parse the date, it is recommended that your timestamps conform to one of the following formats:
+* The first line in the CSV must be a header, containing the names of the fields. 
+* If there is, a field labeled `timestamp` is used for x-axis/time. 
+
+> * The data in the `timestamp` field can be either a number or a string in date-time format. To ensure that the app can parse the date, it is recommended that your timestamps conform to one of the following formats:
  - ISO 8601: `yyyy-mm-dd[T[hh[:mm[:ss[:ms]]]]]`
  - `yyyy-mm-dd [hh[:mm[:ss[:ms]]]]` (Like ISO 8601, but with a space separator for time instead of "T")
  - `mm/dd/yyyy [hh[:mm[:ss[:ms]]]]`
+
 * Note that at this time, the app strips out the first two rows of data. This is because the output of the OPF has three header rows. We are working on a way to automatically detect if the second and third rows are data or meta-data, so this does not happen on non-OPF CSV files.
+* Any modern browser with JavaScript
+* Python for running locally
```

Comment:
We don't actually need Python. Some people may use Python for their web server, some may use node, or Apache. So, Python is not actually a requirement. We could replace this with "Your favorite web server"